src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -196,6 +196,11 @@
     <Compile Include="System\Reflection\TypeInfo.cs" />
     <Compile Include="System\Reflection\TypeInfo.Workaround.cs" />
     <Compile Include="System\Reflection\Runtime\CustomAttributes\RuntimeImplementedCustomAttributeData.cs" />
+    <Compile Include="System\Resources\MissingManifestResourceException.cs" Condition="'$(IsProjectNLibrary)' != 'true'" />
```

Comment:
These are temporary. The matching CoreFX update is required to enable them in ProjectN. I will add TODO next to them.